prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -94,14 +94,15 @@ object FhirPathUtils : Logging {
     fun evaluateCondition(
         appContext: CustomContext?,
         focusResource: Base,
+        contextResource: Base,
```

Comment:
Do you think it would make sense to make `contextResource` an optional parameter that defaults to `bundle`? Could even consider doing the same for `focusResource` and doing similar things for the params in `canEvaluate`. It seems like we have implicit defaults for these parameters, it's just being handled by the callers.

Reply:
I'm hesitant to add default parameters here because I do think callers should consider what value is appropriate for the three values.

Reply:
Ah yea that's fair. Readability-wise on the caller end, I'd prefer to use named parameters to make it clear which is which, especially in cases where we're using the same value for multiple params, but I acknowledge that might just be a personal preference, especially when IDEs help. So I think we're good either way.